packages/theme/components/Hero.vue

```diff
@@ -46,27 +46,41 @@ export default defineComponent({
     },
     background: {
       type: String,
+      default: 'transparent',
+    },
+    image: {
+      type: [Object, String],
       default: '',
     },
-    desktopImage: {
+    imageTag: {
       type: String,
-      default: '',
+      default: null,
```

Comment:
```suggestion
      default: '',
```
('' === null) === false && (typeof null === 'string') === false